src/blocks/VoiceApp.tsx

```diff
@@ -347,7 +443,9 @@ export default class VoiceBlock extends Block {
     )
   }
 
-  // I would like to pass at least a message into this but the block class forbids it...
+  // NOTE: I would like to pass at least a message into this but the block class forbids it...
+  // This method gracefully returns an error to the user, I do not know what it looks like 
+  // because I couldn't get it to render in the testing environment
```

Comment:
good point, we should update the interface for the class so that this function is more permissive and can pass in the parameters you need to render an error